libs/clients/syslumenn/src/lib/syslumennClient.service.ts

```diff
@@ -234,28 +234,38 @@ export class SyslumennService {
 
     const certificate: MortgageCertificate = {
       contentBase64: contentBase64,
+      apiMessage: res.skilabod,
     }
 
     return certificate
   }
 
   async validateMortgageCertificate(
     propertyNumber: string,
+    isFromSearch: boolean | undefined,
   ): Promise<MortgageCertificateValidation> {
     try {
       // Note: this function will throw an error if something goes wrong
       const certificate = await this.getMortgageCertificate(propertyNumber)
 
       const exists = certificate.contentBase64.length !== 0
       const hasKMarking =
-        exists && certificate.contentBase64 !== 'Precondition Required'
+        exists &&
+        certificate.contentBase64 !== 'Precondition Required' &&
+        certificate.apiMessage !== 'Ekki hægt að afgreiða vedbokarvottord'
```

Comment:
messages.ts?

Reply:
I'm going to remove the second line, but the top line needs to compare to exactly 'Precondition Required', so I'm not sure if adding to messages file makes sense for that.

I've been trying to get Sýslumenn to return a boolean (whether mortgage certificate has k marking or not), instead of a string that I have to compare to some specific text, but looks like it is harder than I thought to get it through :/

Reply:
I see. I empathize with you.